### PR TITLE
MRC-1332 rotate keys and audit fixes

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# I believe this is now a default and can be safely removed
```
Comment (vercasson) [inline .travis.yml:1]: deprecated (does nothing now)

File: .travis.yml [hunk 2]
```diff
@@ -21,7 +19,6 @@ cache:
 before_install:
   # Decrypt Google Cloud Storage credentials
   - openssl aes-256-cbc -K $encrypted_9888058ce616_key -iv $encrypted_9888058ce616_iv -in gcs-key.json.enc -out gcs-key.json -d
-# This might be the default these days, but I left it in explicitly
```
Comment (vercasson) [inline .travis.yml:24]: We still need the line above

File: .travis.yml [hunk 3]
```diff
@@ -39,7 +36,6 @@ deploy:
     edge: true
     key_file: gcs-key.json
     bucket: files.meridianapps.com
-    skip_cleanup: true
```
Comment (vercasson) [inline .travis.yml:42]: this is the new default for v2 (always skip cleanup)

File: .travis.yml [hunk 4]
```diff
@@ -48,8 +44,6 @@ deploy:
       node_js: "lts/*"
   # Publish to npm
   - provider: npm
-    # Don't delete `dist/` and `docs/` since we need them to deploy
-    skip_cleanup: true
```
Comment (vercasson) [inline .travis.yml:52]: this is the new default for v2 (always skip cleanup)

File: .travis.yml [hunk 5]
```diff
@@ -58,8 +52,6 @@ deploy:
       node_js: "lts/*"
   # Update GitHub pages for the documentation and demos
   - provider: pages
-    # Don't delete `dist/` and `docs/` since we need them to deploy
-    skip_cleanup: true
```
Comment (vercasson) [inline .travis.yml:62]: this is the new default for v2 (always skip cleanup)

**Comment by vercasson** [comment]
I also updated this doc regarding "Google Cloud". I added info on the Project, Service Account, and GCS Bucket. I had to really hunt down the Project and Service Account. We put this under "meridian docs" which surprised me.

**Comment by tarakreft** [comment]
There's a package-lock.json change funny enough. Is this intentional?

**Comment by vercasson** [comment]
> There's a package-lock.json change funny enough. Is this intentional?

Yup, I updated a package (audit fix)

**Comment by tarakreft** [comment]
> > There's a package-lock.json change funny enough. Is this intentional?
> 
> Yup, I updated a package (audit fix)

Oh interesting -- wouldn't there need to be a corresponding change in `package.json`?

**Comment by vercasson** [comment]
> > > There's a package-lock.json change funny enough. Is this intentional?
> > 
> > 
> > Yup, I updated a package (audit fix)
> 
> Oh interesting -- wouldn't there need to be a corresponding change in `package.json`?

For audit fixes that's not always the case. It does seem odd, but the diff sorta tells the story. :)

**Review by tarakreft — APPROVED** [review]

**Pull request merged**